CMakeLists.txt

```diff
@@ -241,7 +241,7 @@ macro(xeus_r_create_target target_name linkage output_name)
     endif ()
 
     if (EMSCRIPTEN)
-        target_link_libraries(${target_name} PUBLIC ${XEUS_R_XEUS_TARGET} PRIVATE ${R_LIBRARY_BASE} ${R_LIBRARY_BLAS})
+        target_link_libraries(${target_name} PUBLIC ${XEUS_R_XEUS_TARGET} PRIVATE ${R_LIBRARY_BASE} ${R_LIBRARY_BLAS} ${R_LDFLAGS})
```

Comment:
Also need to add to wasm-host environment
```
  - pcre2
  - zlib
  - libflang
  - xz
  - bzip2
  - libiconv
```

Reply:
Ahh yes correct. So technically these end up in $PREFIX/lib i think !